lib/cli.rb

```diff
@@ -18,7 +18,22 @@
 class Cli
   extend GLI::App
 
-  program_desc 'A tool for running integration tests inside a network of virtual machines'
+  program_desc 'A tool for controlling networks of machines for integration testing'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.